src/components/NcListItem/NcListItem.vue

```diff
@@ -672,8 +711,14 @@ export default {
 		color: var(--color-text-lighter);
 	}
 
-	&__counter {
-		margin: 2px 4px 0 4px;
+	&__additional_elements {
+		margin-top: 5px;
```

Comment:
If we change this, items without indicator start jumping on hover.
```suggestion
		margin: 2px 4px 0 4px;
```

Reply:
i could't detect it, but will try to apply your solution

Reply:
> i could't detect it, but will try to apply your solution

Happens for me in the docs when hovering the first ListItem in the example both on Firefox 103 and Chrome 104. The text moves slightly downwards.